core/src/main/java/jenkins/triggers/SCMTriggerItem.java

```diff
@@ -116,6 +120,11 @@
                 return delegate.asProject().scheduleBuild2(quietPeriod, null, actions);
             }
             @Override public PollingResult poll(TaskListener listener) {
+                SCMDecisionHandler veto = SCMDecisionHandler.firstShouldPollVeto(asItem());
+                if (!veto.shouldPoll(asItem())) {
```

Comment:
@stephenc This is missing a `null` check, resulting in [JENKINS-36232](https://issues.jenkins-ci.org/browse/JENKINS-36232). Could you PTAL?